watchtower/wtwire/session_init_reply.go

```diff
@@ -0,0 +1,62 @@
+package wtwire
+
+import "io"
+
+// TODO(conner): enumerate fail codes
+
+// SessionInitReply is a message sent from watchtower to client in response to a
+// SessionInit message, and signals either an acceptance or rejection of the
+// proposed session parameters.
+type SessionInitReply struct {
+	// Code will be non-zero if the watchtower rejected the session init.
+	Code uint16
+
+	// RewardPkScript is the reward address to be included in any sweep
```

Comment:
FWIW, the PR description for this PR doesn't match this particular field. Is the idea that that code returned will determine if this is an opaque payload, and how to interpret it? So for example, if a private tower, then it's blank, while if it's a public tower it may also have a reward script and also payment details.